src/App/Pages/Generator/GeneratorPage.xaml.cs

```diff
@@ -15,17 +16,21 @@ public partial class GeneratorPage : BaseContentPage, IThemeDirtablePage
 
         private GeneratorPageViewModel _vm;
         private readonly bool _fromTabPage;
+        private readonly bool _fromExtension;
```

Comment:
Instead of this flag, I'd use the same approach as in other places where we use `AppOptions` and in it you find the flag